cli/running/watchdog.go

```diff
@@ -123,6 +116,13 @@ func (wd *Watchdog) startSignalHandling() {
 		syscall.SIGTERM, syscall.SIGHUP)
 	signal.Notify(sigChan)
 
+	// This call is needed to ignore SIGURG signals which are part of
+	// preemptive multitasking implementation in go. See:
+	// https://go.googlesource.com/proposal/+/master/design/24543-non-cooperative-preemption.md.
+	// Also, there is no way to detect if a signal is related to the runtime or not:
+	// https://github.com/golang/go/issues/37942.
+	signal.Ignore(syscall.SIGURG)
```

Comment:
Did you check this does not affect a sig mask of the child Tarantool process?

Reply:
The child process is already started by this moment. It will not affect it.

Reply:
Oh, yes... you've moved start handling again :)